kubejs/client_scripts/tooltips/dyeing.js

```diff
@@ -12,3 +12,9 @@ addTooltip(
   ["copper_ingot", "raw_copper"],
   "Can be made by dyeing _orange powder_ or _stone_ with _orange dye_"
 );
+
+addTooltip(["deepslate_diamond_ore"], "Can be made by dyeing _deepslate_ with _cyan dye_");
```

Comment:
I think it's light blue, not cyan

Reply:
![image](https://github.com/game-design-driven/Create-Prepare-to-Dye/assets/81585596/7f2cfa5c-08fc-47d5-8631-1526da2ceeb5)
in-game its cyan dye.

Reply:
oh I see. that's a bug then. should be light blue

Reply:
i just finished automating cyan dye *cry*

Will change

Reply:
would be funny to use cyan to craft light blue picaxes